[2023-02-27T09:32:51.238+0000] {processor.py:153} INFO - Started process (PID=4099) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:32:51.239+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:32:51.239+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:32:51.239+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:32:51.250+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:32:51.249+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 7, in <module>
    default_args=default_args, catchup=False
NameError: name 'default_args' is not defined
[2023-02-27T09:32:51.250+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:32:51.275+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.039 seconds
[2023-02-27T09:33:21.562+0000] {processor.py:153} INFO - Started process (PID=4128) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:33:21.564+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:33:21.565+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:33:21.565+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:33:21.577+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:33:21.575+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 7, in <module>
    default_args=default_args, catchup=False
NameError: name 'default_args' is not defined
[2023-02-27T09:33:21.577+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:33:21.595+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.038 seconds
[2023-02-27T09:33:40.791+0000] {processor.py:153} INFO - Started process (PID=4154) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:33:40.795+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:33:40.798+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:33:40.798+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:33:40.836+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:33:40.833+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 14, in <module>
    mode='reschedule'
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 377, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'filepath'
[2023-02-27T09:33:40.836+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:33:40.858+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.078 seconds
[2023-02-27T09:34:11.168+0000] {processor.py:153} INFO - Started process (PID=4187) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:34:11.169+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:34:11.171+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:34:11.170+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:34:11.190+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:34:11.188+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 14, in <module>
    mode='reschedule'
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 377, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'filepath'
[2023-02-27T09:34:11.191+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:34:11.210+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.046 seconds
[2023-02-27T09:34:15.241+0000] {processor.py:153} INFO - Started process (PID=4193) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:34:15.243+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:34:15.244+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:34:15.244+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:34:15.275+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:34:15.272+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 11
    
    ^
SyntaxError: unexpected EOF while parsing
[2023-02-27T09:34:15.278+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:34:15.344+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.108 seconds
[2023-02-27T09:34:18.302+0000] {processor.py:153} INFO - Started process (PID=4194) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:34:18.304+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:34:18.306+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:34:18.306+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:34:18.315+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:34:18.314+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 5
    default_args =
                  ^
SyntaxError: invalid syntax
[2023-02-27T09:34:18.317+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:34:18.343+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.045 seconds
[2023-02-27T09:34:30.464+0000] {processor.py:153} INFO - Started process (PID=4208) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:34:30.466+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:34:30.467+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:34:30.467+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:34:30.474+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:34:30.473+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 13
    
    ^
SyntaxError: unexpected EOF while parsing
[2023-02-27T09:34:30.475+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:34:30.496+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.038 seconds
[2023-02-27T09:34:40.631+0000] {processor.py:153} INFO - Started process (PID=4221) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:34:40.633+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:34:40.634+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:34:40.634+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:34:40.642+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:34:40.641+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 13
    
    ^
SyntaxError: unexpected EOF while parsing
[2023-02-27T09:34:40.643+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:34:40.670+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.047 seconds
[2023-02-27T09:34:53.796+0000] {processor.py:153} INFO - Started process (PID=4234) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:34:53.798+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:34:53.800+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:34:53.799+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:34:53.807+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:34:53.806+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 14
    
    ^
SyntaxError: unexpected EOF while parsing
[2023-02-27T09:34:53.809+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:34:53.834+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.044 seconds
[2023-02-27T09:35:03.960+0000] {processor.py:153} INFO - Started process (PID=4248) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:35:03.962+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:35:03.963+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:35:03.963+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:35:03.969+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:35:03.968+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 14
    
    ^
SyntaxError: unexpected EOF while parsing
[2023-02-27T09:35:03.970+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:35:03.993+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.036 seconds
[2023-02-27T09:35:25.153+0000] {processor.py:153} INFO - Started process (PID=4273) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:35:25.155+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:35:25.157+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:35:25.157+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:35:25.166+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:35:25.164+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 8
    }
    ^
SyntaxError: invalid syntax
[2023-02-27T09:35:25.167+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:35:25.187+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.040 seconds
[2023-02-27T09:35:43.388+0000] {processor.py:153} INFO - Started process (PID=4287) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:35:43.390+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:35:43.391+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:35:43.391+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:35:43.397+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:35:43.396+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 14
    
    ^
SyntaxError: unexpected EOF while parsing
[2023-02-27T09:35:43.398+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:35:43.422+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.036 seconds
[2023-02-27T09:35:56.586+0000] {processor.py:153} INFO - Started process (PID=4310) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:35:56.593+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:35:56.598+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:35:56.598+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:35:56.669+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:35:56.667+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 15
    
    ^
SyntaxError: unexpected EOF while parsing
[2023-02-27T09:35:56.670+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:35:56.705+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.128 seconds
[2023-02-27T09:36:15.779+0000] {processor.py:153} INFO - Started process (PID=4327) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:36:15.781+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:36:15.782+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:36:15.782+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:36:15.793+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:36:15.789+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 15
    
    ^
SyntaxError: unexpected EOF while parsing
[2023-02-27T09:36:15.795+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:36:15.835+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.062 seconds
[2023-02-27T09:36:34.023+0000] {processor.py:153} INFO - Started process (PID=4353) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:36:34.025+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:36:34.026+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:36:34.026+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:36:34.033+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:36:34.032+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 15
    
    ^
SyntaxError: unexpected EOF while parsing
[2023-02-27T09:36:34.035+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:36:34.060+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.041 seconds
[2023-02-27T09:37:03.374+0000] {processor.py:153} INFO - Started process (PID=4380) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:37:03.376+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:37:03.378+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:37:03.378+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:37:03.387+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:37:03.386+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 16
    sensor =
            ^
SyntaxError: invalid syntax
[2023-02-27T09:37:03.388+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:37:03.433+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.065 seconds
[2023-02-27T09:37:29.902+0000] {processor.py:153} INFO - Started process (PID=4418) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:37:29.904+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:37:29.905+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:37:29.905+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:37:29.927+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:37:29.922+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 17, in <module>
    sensor = ExternalTaskSensor(
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 377, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'external_dag_id'
[2023-02-27T09:37:29.929+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:37:29.954+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.056 seconds
[2023-02-27T09:37:36.967+0000] {processor.py:153} INFO - Started process (PID=4419) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:37:36.969+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:37:36.970+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:37:36.970+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:37:36.996+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:37:36.992+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 18, in <module>
    task_id=''
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 377, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'external_dag_id'
[2023-02-27T09:37:36.999+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:37:37.025+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.062 seconds
[2023-02-27T09:37:45.167+0000] {processor.py:153} INFO - Started process (PID=4432) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:37:45.171+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:37:45.173+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:37:45.173+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:37:45.223+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:37:45.220+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 18, in <module>
    task_id='sensor',
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 377, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'external_dag_id'
[2023-02-27T09:37:45.224+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:37:45.245+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.084 seconds
[2023-02-27T09:38:12.513+0000] {processor.py:153} INFO - Started process (PID=4458) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:38:12.516+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:38:12.520+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:38:12.520+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:38:12.560+0000] {processor.py:753} INFO - DAG(s) dict_keys(['master_dag']) retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:38:12.677+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:38:12.677+0000] {manager.py:504} INFO - Created Permission View: can read on DAG:master_dag
[2023-02-27T09:38:12.682+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:38:12.682+0000] {manager.py:504} INFO - Created Permission View: can edit on DAG:master_dag
[2023-02-27T09:38:12.686+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:38:12.686+0000] {manager.py:504} INFO - Created Permission View: can delete on DAG:master_dag
[2023-02-27T09:38:12.687+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:38:12.686+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-27T09:38:12.695+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:38:12.695+0000] {dag.py:2711} INFO - Creating ORM DAG for master_dag
[2023-02-27T09:38:12.704+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:38:12.703+0000] {dag.py:3441} INFO - Setting next_dagrun for master_dag to 2023-02-26T00:00:00+00:00, run_after=2023-02-27T00:00:00+00:00
[2023-02-27T09:38:12.716+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.207 seconds
[2023-02-27T09:38:43.037+0000] {processor.py:153} INFO - Started process (PID=4495) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:38:43.039+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:38:43.040+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:38:43.040+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:38:43.063+0000] {processor.py:753} INFO - DAG(s) dict_keys(['master_dag']) retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:38:43.086+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:38:43.086+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-27T09:38:43.103+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:38:43.103+0000] {dag.py:3441} INFO - Setting next_dagrun for master_dag to 2023-02-26T00:00:00+00:00, run_after=2023-02-27T00:00:00+00:00
[2023-02-27T09:38:43.115+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.081 seconds
[2023-02-27T09:39:13.428+0000] {processor.py:153} INFO - Started process (PID=4533) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:39:13.430+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:39:13.432+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:39:13.432+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:39:13.457+0000] {processor.py:753} INFO - DAG(s) dict_keys(['master_dag']) retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:39:13.476+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:39:13.476+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-27T09:39:13.493+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:39:13.493+0000] {dag.py:3441} INFO - Setting next_dagrun for master_dag to 2023-02-26T00:00:00+00:00, run_after=2023-02-27T00:00:00+00:00
[2023-02-27T09:39:13.504+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.081 seconds
[2023-02-27T09:39:43.842+0000] {processor.py:153} INFO - Started process (PID=4559) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:39:43.845+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:39:43.846+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:39:43.846+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:39:43.869+0000] {processor.py:753} INFO - DAG(s) dict_keys(['master_dag']) retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:39:43.888+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:39:43.888+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-27T09:39:43.903+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:39:43.902+0000] {dag.py:3441} INFO - Setting next_dagrun for master_dag to 2023-02-26T00:00:00+00:00, run_after=2023-02-27T00:00:00+00:00
[2023-02-27T09:39:43.912+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.078 seconds
[2023-02-27T09:40:14.232+0000] {processor.py:153} INFO - Started process (PID=4600) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:40:14.235+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:40:14.236+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:40:14.236+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:40:14.256+0000] {processor.py:753} INFO - DAG(s) dict_keys(['master_dag']) retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:40:14.276+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:40:14.276+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-27T09:40:14.291+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:40:14.291+0000] {dag.py:3441} INFO - Setting next_dagrun for master_dag to 2023-02-26T00:00:00+00:00, run_after=2023-02-27T00:00:00+00:00
[2023-02-27T09:40:14.301+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.075 seconds
[2023-02-27T09:40:44.557+0000] {processor.py:153} INFO - Started process (PID=4638) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:40:44.561+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:40:44.563+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:40:44.563+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:40:44.587+0000] {processor.py:753} INFO - DAG(s) dict_keys(['master_dag']) retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:40:44.606+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:40:44.606+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-27T09:40:44.621+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:40:44.621+0000] {dag.py:3441} INFO - Setting next_dagrun for master_dag to 2023-02-26T00:00:00+00:00, run_after=2023-02-27T00:00:00+00:00
[2023-02-27T09:40:44.631+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.080 seconds
[2023-02-27T09:41:14.891+0000] {processor.py:153} INFO - Started process (PID=4664) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:41:14.894+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:41:14.895+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:41:14.895+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:41:14.922+0000] {processor.py:753} INFO - DAG(s) dict_keys(['master_dag']) retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:41:14.944+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:41:14.944+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-27T09:41:14.959+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:41:14.959+0000] {dag.py:3441} INFO - Setting next_dagrun for master_dag to 2023-02-26T00:00:00+00:00, run_after=2023-02-27T00:00:00+00:00
[2023-02-27T09:41:14.970+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.084 seconds
[2023-02-27T09:41:45.238+0000] {processor.py:153} INFO - Started process (PID=4701) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:41:45.240+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:41:45.242+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:41:45.241+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:41:45.263+0000] {processor.py:753} INFO - DAG(s) dict_keys(['master_dag']) retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:41:45.285+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:41:45.285+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-27T09:41:45.304+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:41:45.304+0000] {dag.py:3441} INFO - Setting next_dagrun for master_dag to 2023-02-26T00:00:00+00:00, run_after=2023-02-27T00:00:00+00:00
[2023-02-27T09:41:45.318+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.089 seconds
[2023-02-27T09:41:53.384+0000] {processor.py:153} INFO - Started process (PID=4714) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:41:53.392+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:41:53.394+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:41:53.394+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:41:53.404+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:41:53.402+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 23
    last_task =
               ^
SyntaxError: invalid syntax
[2023-02-27T09:41:53.406+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:41:53.446+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.069 seconds
[2023-02-27T09:42:15.815+0000] {processor.py:153} INFO - Started process (PID=4740) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:42:15.818+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:42:15.821+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:42:15.821+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:42:15.832+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:42:15.830+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 3
    from airflow.sensors.d
                         ^
SyntaxError: invalid syntax
[2023-02-27T09:42:15.832+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:42:15.863+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.085 seconds
[2023-02-27T09:42:43.159+0000] {processor.py:153} INFO - Started process (PID=4766) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:42:43.162+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:42:43.163+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:42:43.163+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:42:43.172+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:42:43.170+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 3
    from airflow.operators.dummy_
                                ^
SyntaxError: invalid syntax
[2023-02-27T09:42:43.173+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:42:43.206+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.052 seconds
[2023-02-27T09:43:13.526+0000] {processor.py:153} INFO - Started process (PID=4792) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:43:13.530+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:43:13.532+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:43:13.532+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:43:13.539+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:43:13.538+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 3
    from airflow.operators.dummy_
                                ^
SyntaxError: invalid syntax
[2023-02-27T09:43:13.540+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:43:13.567+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.048 seconds
[2023-02-27T09:43:23.694+0000] {processor.py:153} INFO - Started process (PID=4805) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:43:23.695+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:43:23.697+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:43:23.696+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:43:23.711+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:43:23.709+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 3, in <module>
    from airflow.operators.dummy_operator import D
ImportError: cannot import name 'D' from 'airflow.operators.dummy_operator' (unknown location)
[2023-02-27T09:43:23.712+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:43:23.734+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.043 seconds
[2023-02-27T09:43:49.054+0000] {processor.py:153} INFO - Started process (PID=4831) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:43:49.058+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:43:49.059+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:43:49.059+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:43:49.071+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:43:49.069+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 3
    from airflow.operators.bash import
                                      ^
SyntaxError: invalid syntax
[2023-02-27T09:43:49.072+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:43:49.114+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.064 seconds
[2023-02-27T09:44:15.444+0000] {processor.py:153} INFO - Started process (PID=4869) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:44:15.447+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:44:15.449+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:44:15.448+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:44:15.464+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:44:15.463+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 3
    from airflow.operators.bash_operator import
                                              ^
SyntaxError: invalid syntax
[2023-02-27T09:44:15.466+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:44:15.489+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.051 seconds
[2023-02-27T09:44:45.848+0000] {processor.py:153} INFO - Started process (PID=4895) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:44:45.853+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:44:45.855+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:44:45.855+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:44:45.864+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:44:45.863+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 3
    from airflow.operators.bash_operator import
                                              ^
SyntaxError: invalid syntax
[2023-02-27T09:44:45.865+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:44:45.890+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.050 seconds
[2023-02-27T09:44:55.994+0000] {processor.py:153} INFO - Started process (PID=4908) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:44:55.996+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:44:55.997+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:44:55.997+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:44:56.017+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:44:56.014+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 4, in <module>
    from airflow.operators.empty import Du
ImportError: cannot import name 'Du' from 'airflow.operators.empty' (/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/empty.py)
[2023-02-27T09:44:56.018+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:44:56.047+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.057 seconds
[2023-02-27T09:45:05.122+0000] {processor.py:153} INFO - Started process (PID=4921) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:45:05.124+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:45:05.126+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:45:05.125+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:45:05.150+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:45:05.147+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 4, in <module>
    from airflow.operators.empty import DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.empty' (/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/empty.py)
[2023-02-27T09:45:05.151+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:45:05.178+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.063 seconds
[2023-02-27T09:45:35.447+0000] {processor.py:153} INFO - Started process (PID=4947) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:45:35.449+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:45:35.451+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:45:35.451+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:45:35.462+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:45:35.461+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 4, in <module>
    from airflow.operators.empty import DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.empty' (/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/empty.py)
[2023-02-27T09:45:35.463+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:45:35.483+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.040 seconds
[2023-02-27T09:46:05.781+0000] {processor.py:153} INFO - Started process (PID=4985) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:46:05.782+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:46:05.783+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:46:05.783+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:46:05.792+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:46:05.790+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 4, in <module>
    from airflow.operators.empty import DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.empty' (/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/empty.py)
[2023-02-27T09:46:05.793+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:46:05.812+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.034 seconds
[2023-02-27T09:46:36.098+0000] {processor.py:153} INFO - Started process (PID=5023) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:46:36.103+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:46:36.104+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:46:36.104+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:46:36.113+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:46:36.111+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 4, in <module>
    from airflow.operators.empty import DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.empty' (/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/empty.py)
[2023-02-27T09:46:36.114+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:46:36.133+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.038 seconds
[2023-02-27T09:47:06.498+0000] {processor.py:153} INFO - Started process (PID=5049) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:47:06.501+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:47:06.503+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:47:06.502+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:47:06.517+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:47:06.515+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 4, in <module>
    from airflow.operators.empty import DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.empty' (/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/empty.py)
[2023-02-27T09:47:06.518+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:47:06.542+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.049 seconds
[2023-02-27T09:47:24.790+0000] {processor.py:153} INFO - Started process (PID=5075) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:47:24.793+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:47:24.794+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:47:24.794+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:47:24.812+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:47:24.810+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 4, in <module>
    from airflow.operators.python import D
ImportError: cannot import name 'D' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py)
[2023-02-27T09:47:24.813+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:47:24.835+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.050 seconds
[2023-02-27T09:47:55.169+0000] {processor.py:153} INFO - Started process (PID=5101) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:47:55.172+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:47:55.173+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:47:55.173+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:47:55.184+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:47:55.182+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 4, in <module>
    from airflow.operators.python import D
ImportError: cannot import name 'D' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py)
[2023-02-27T09:47:55.186+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:47:55.208+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.043 seconds
[2023-02-27T09:48:25.532+0000] {processor.py:153} INFO - Started process (PID=5139) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:48:25.534+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:48:25.535+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:48:25.535+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:48:25.545+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:48:25.544+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 4, in <module>
    from airflow.operators.python import D
ImportError: cannot import name 'D' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py)
[2023-02-27T09:48:25.546+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:48:25.568+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.040 seconds
[2023-02-27T09:48:55.850+0000] {processor.py:153} INFO - Started process (PID=5177) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:48:55.853+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:48:55.854+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:48:55.854+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:48:55.865+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:48:55.863+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 4, in <module>
    from airflow.operators.python import D
ImportError: cannot import name 'D' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py)
[2023-02-27T09:48:55.865+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:48:55.889+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.042 seconds
[2023-02-27T09:49:26.165+0000] {processor.py:153} INFO - Started process (PID=5202) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:49:26.169+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:49:26.171+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:49:26.171+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:49:26.188+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:49:26.186+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 4, in <module>
    from airflow.operators.python import D
ImportError: cannot import name 'D' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py)
[2023-02-27T09:49:26.190+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:49:26.219+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.060 seconds
[2023-02-27T09:49:56.625+0000] {processor.py:153} INFO - Started process (PID=5240) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:49:56.627+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:49:56.629+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:49:56.629+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:49:56.644+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:49:56.642+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 4, in <module>
    from airflow.operators.python import D
ImportError: cannot import name 'D' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py)
[2023-02-27T09:49:56.645+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:49:56.665+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.046 seconds
[2023-02-27T09:50:26.966+0000] {processor.py:153} INFO - Started process (PID=5266) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:50:26.969+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:50:26.970+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:50:26.970+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:50:26.983+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:50:26.981+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 4, in <module>
    from airflow.operators.python import D
ImportError: cannot import name 'D' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py)
[2023-02-27T09:50:26.984+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:50:27.010+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.050 seconds
[2023-02-27T09:50:57.240+0000] {processor.py:153} INFO - Started process (PID=5303) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:50:57.242+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:50:57.244+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:50:57.243+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:50:57.256+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:50:57.255+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 4, in <module>
    from airflow.operators.python import D
ImportError: cannot import name 'D' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py)
[2023-02-27T09:50:57.257+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:50:57.275+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.041 seconds
[2023-02-27T09:51:27.597+0000] {processor.py:153} INFO - Started process (PID=5336) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:51:27.599+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:51:27.600+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:51:27.600+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:51:27.612+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:51:27.610+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 4, in <module>
    from airflow.operators.python import D
ImportError: cannot import name 'D' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py)
[2023-02-27T09:51:27.613+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:51:27.635+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.042 seconds
[2023-02-27T09:51:31.717+0000] {processor.py:153} INFO - Started process (PID=5342) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:51:31.719+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:51:31.720+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:51:31.720+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:51:31.728+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:51:31.726+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 4
    from airflow.operators.empty import
                                       ^
SyntaxError: invalid syntax
[2023-02-27T09:51:31.729+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:51:31.753+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.040 seconds
[2023-02-27T09:52:02.188+0000] {processor.py:153} INFO - Started process (PID=5367) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:52:02.191+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:52:02.192+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:52:02.192+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:52:02.202+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:52:02.200+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 4
    from airflow.operators.empty import
                                       ^
SyntaxError: invalid syntax
[2023-02-27T09:52:02.203+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:52:02.224+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.043 seconds
[2023-02-27T09:52:32.631+0000] {processor.py:153} INFO - Started process (PID=5405) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:52:32.634+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:52:32.636+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:52:32.635+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:52:32.644+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:52:32.642+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 4
    from airflow.operators.empty import
                                       ^
SyntaxError: invalid syntax
[2023-02-27T09:52:32.645+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:52:32.668+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.046 seconds
[2023-02-27T09:53:03.070+0000] {processor.py:153} INFO - Started process (PID=5444) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:53:03.072+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:53:03.074+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:53:03.074+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:53:03.080+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:53:03.078+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 4
    from airflow.operators.empty import
                                       ^
SyntaxError: invalid syntax
[2023-02-27T09:53:03.082+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:53:03.106+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.039 seconds
[2023-02-27T09:53:33.444+0000] {processor.py:153} INFO - Started process (PID=5471) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:53:33.448+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:53:33.450+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:53:33.450+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:53:33.457+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:53:33.455+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 4
    from airflow.operators.empty import
                                       ^
SyntaxError: invalid syntax
[2023-02-27T09:53:33.458+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:53:33.480+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.045 seconds
[2023-02-27T09:54:03.794+0000] {processor.py:153} INFO - Started process (PID=5508) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:54:03.795+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:54:03.796+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:54:03.796+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:54:03.807+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:54:03.806+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 4
    from airflow.operators.empty import
                                       ^
SyntaxError: invalid syntax
[2023-02-27T09:54:03.808+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:54:03.827+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.037 seconds
[2023-02-27T09:54:34.144+0000] {processor.py:153} INFO - Started process (PID=5533) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:54:34.147+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:54:34.148+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:54:34.148+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:54:34.160+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:54:34.155+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 4
    from airflow.operators.empty import
                                       ^
SyntaxError: invalid syntax
[2023-02-27T09:54:34.162+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:54:34.197+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.059 seconds
[2023-02-27T09:55:04.539+0000] {processor.py:153} INFO - Started process (PID=5571) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:55:04.543+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:55:04.545+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:55:04.545+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:55:04.554+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:55:04.553+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 4
    from airflow.operators.empty import
                                       ^
SyntaxError: invalid syntax
[2023-02-27T09:55:04.556+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:55:04.584+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.050 seconds
[2023-02-27T09:55:34.922+0000] {processor.py:153} INFO - Started process (PID=5606) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:55:34.923+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:55:34.924+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:55:34.924+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:55:34.930+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:55:34.929+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 4
    from airflow.operators.empty import
                                       ^
SyntaxError: invalid syntax
[2023-02-27T09:55:34.930+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:55:34.948+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.029 seconds
[2023-02-27T09:56:05.286+0000] {processor.py:153} INFO - Started process (PID=5634) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:56:05.290+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:56:05.292+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:56:05.291+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:56:05.300+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:56:05.298+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 4
    from airflow.operators.empty import
                                       ^
SyntaxError: invalid syntax
[2023-02-27T09:56:05.301+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:56:05.324+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.046 seconds
[2023-02-27T09:56:35.698+0000] {processor.py:153} INFO - Started process (PID=5672) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:56:35.701+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:56:35.703+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:56:35.703+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:56:35.711+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:56:35.709+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 4
    from airflow.operators.empty import
                                       ^
SyntaxError: invalid syntax
[2023-02-27T09:56:35.712+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:56:35.741+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.051 seconds
[2023-02-27T09:56:36.698+0000] {processor.py:153} INFO - Started process (PID=5673) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:56:36.700+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:56:36.701+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:56:36.701+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:56:36.739+0000] {processor.py:753} INFO - DAG(s) dict_keys(['master_dag']) retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:56:36.799+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:56:36.799+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-27T09:56:36.822+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:56:36.821+0000] {dag.py:3441} INFO - Setting next_dagrun for master_dag to 2023-02-26T00:00:00+00:00, run_after=2023-02-27T00:00:00+00:00
[2023-02-27T09:56:36.843+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.149 seconds
[2023-02-27T09:57:07.196+0000] {processor.py:153} INFO - Started process (PID=5699) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:57:07.198+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:57:07.200+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:57:07.200+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:57:07.220+0000] {processor.py:753} INFO - DAG(s) dict_keys(['master_dag']) retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:57:07.242+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:57:07.242+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-27T09:57:07.262+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:57:07.261+0000] {dag.py:3441} INFO - Setting next_dagrun for master_dag to 2023-02-26T00:00:00+00:00, run_after=2023-02-27T00:00:00+00:00
[2023-02-27T09:57:07.273+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.080 seconds
[2023-02-27T09:57:27.561+0000] {processor.py:153} INFO - Started process (PID=5725) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:57:27.565+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:57:27.567+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:57:27.566+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:57:27.609+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:57:27.606+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 24, in <module>
    last_task = EmptyOperator(
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 394, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 394, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
TypeError: __init__() missing 1 required positional argument: 'task_id'
[2023-02-27T09:57:27.610+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:57:27.641+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.097 seconds
[2023-02-27T09:57:37.807+0000] {processor.py:153} INFO - Started process (PID=5738) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:57:37.809+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:57:37.811+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:57:37.811+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:57:37.873+0000] {processor.py:753} INFO - DAG(s) dict_keys(['master_dag']) retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:57:37.945+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:57:37.945+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-27T09:57:37.964+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:57:37.964+0000] {dag.py:3441} INFO - Setting next_dagrun for master_dag to 2023-02-26T00:00:00+00:00, run_after=2023-02-27T00:00:00+00:00
[2023-02-27T09:57:38.003+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.215 seconds
[2023-02-27T09:57:47.941+0000] {processor.py:153} INFO - Started process (PID=5752) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:57:47.943+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:57:47.945+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:57:47.945+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:57:47.993+0000] {processor.py:753} INFO - DAG(s) dict_keys(['master_dag']) retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:57:48.022+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:57:48.021+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-27T09:57:48.051+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:57:48.051+0000] {dag.py:3441} INFO - Setting next_dagrun for master_dag to 2023-02-26T00:00:00+00:00, run_after=2023-02-27T00:00:00+00:00
[2023-02-27T09:57:48.073+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.139 seconds
[2023-02-27T09:58:18.347+0000] {processor.py:153} INFO - Started process (PID=5778) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:58:18.350+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:58:18.352+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:58:18.352+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:58:18.380+0000] {processor.py:753} INFO - DAG(s) dict_keys(['master_dag']) retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:58:18.427+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:58:18.427+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-27T09:58:18.443+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:58:18.443+0000] {dag.py:3441} INFO - Setting next_dagrun for master_dag to 2023-02-26T00:00:00+00:00, run_after=2023-02-27T00:00:00+00:00
[2023-02-27T09:58:18.461+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.121 seconds
[2023-02-27T09:58:48.676+0000] {processor.py:153} INFO - Started process (PID=5816) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:58:48.678+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:58:48.679+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:58:48.679+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:58:48.697+0000] {processor.py:753} INFO - DAG(s) dict_keys(['master_dag']) retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:58:48.721+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:58:48.721+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-27T09:58:48.740+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:58:48.740+0000] {dag.py:3441} INFO - Setting next_dagrun for master_dag to 2023-02-26T00:00:00+00:00, run_after=2023-02-27T00:00:00+00:00
[2023-02-27T09:58:48.756+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.083 seconds
[2023-02-27T09:59:19.065+0000] {processor.py:153} INFO - Started process (PID=5854) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:59:19.068+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:59:19.071+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:59:19.070+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:59:19.099+0000] {processor.py:753} INFO - DAG(s) dict_keys(['master_dag']) retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:59:19.138+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:59:19.138+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-27T09:59:19.165+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:59:19.165+0000] {dag.py:3441} INFO - Setting next_dagrun for master_dag to 2023-02-26T00:00:00+00:00, run_after=2023-02-27T00:00:00+00:00
[2023-02-27T09:59:19.182+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.124 seconds
[2023-02-27T09:59:49.502+0000] {processor.py:153} INFO - Started process (PID=5880) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T09:59:49.505+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T09:59:49.506+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:59:49.506+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T09:59:49.527+0000] {processor.py:753} INFO - DAG(s) dict_keys(['master_dag']) retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T09:59:49.552+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:59:49.551+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-27T09:59:49.573+0000] {logging_mixin.py:137} INFO - [2023-02-27T09:59:49.573+0000] {dag.py:3441} INFO - Setting next_dagrun for master_dag to 2023-02-26T00:00:00+00:00, run_after=2023-02-27T00:00:00+00:00
[2023-02-27T09:59:49.589+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.091 seconds
[2023-02-27T10:00:19.921+0000] {processor.py:153} INFO - Started process (PID=5919) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T10:00:19.923+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T10:00:19.925+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:00:19.924+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T10:00:19.945+0000] {processor.py:753} INFO - DAG(s) dict_keys(['master_dag']) retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T10:00:19.969+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:00:19.969+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-27T10:00:19.988+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:00:19.988+0000] {dag.py:3441} INFO - Setting next_dagrun for master_dag to 2023-02-26T00:00:00+00:00, run_after=2023-02-27T00:00:00+00:00
[2023-02-27T10:00:20.001+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.084 seconds
[2023-02-27T10:00:50.474+0000] {processor.py:153} INFO - Started process (PID=5944) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T10:00:50.475+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T10:00:50.476+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:00:50.476+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T10:00:50.494+0000] {processor.py:753} INFO - DAG(s) dict_keys(['master_dag']) retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T10:00:50.517+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:00:50.517+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-27T10:00:50.536+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:00:50.536+0000] {dag.py:3441} INFO - Setting next_dagrun for master_dag to 2023-02-26T00:00:00+00:00, run_after=2023-02-27T00:00:00+00:00
[2023-02-27T10:00:50.551+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.080 seconds
[2023-02-27T10:01:20.838+0000] {processor.py:153} INFO - Started process (PID=5982) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T10:01:20.841+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T10:01:20.842+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:01:20.842+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T10:01:20.873+0000] {processor.py:753} INFO - DAG(s) dict_keys(['master_dag']) retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T10:01:20.897+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:01:20.897+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-27T10:01:20.917+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:01:20.917+0000] {dag.py:3441} INFO - Setting next_dagrun for master_dag to 2023-02-26T00:00:00+00:00, run_after=2023-02-27T00:00:00+00:00
[2023-02-27T10:01:20.931+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.102 seconds
[2023-02-27T10:01:51.373+0000] {processor.py:153} INFO - Started process (PID=6008) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T10:01:51.375+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T10:01:51.377+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:01:51.377+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T10:01:51.395+0000] {processor.py:753} INFO - DAG(s) dict_keys(['master_dag']) retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T10:01:51.420+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:01:51.420+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-27T10:01:51.442+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:01:51.442+0000] {dag.py:3441} INFO - Setting next_dagrun for master_dag to 2023-02-26T00:00:00+00:00, run_after=2023-02-27T00:00:00+00:00
[2023-02-27T10:01:51.456+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.087 seconds
[2023-02-27T10:02:21.733+0000] {processor.py:153} INFO - Started process (PID=6046) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T10:02:21.735+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T10:02:21.736+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:02:21.736+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T10:02:21.759+0000] {processor.py:753} INFO - DAG(s) dict_keys(['master_dag']) retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T10:02:21.785+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:02:21.785+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-27T10:02:21.805+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:02:21.805+0000] {dag.py:3441} INFO - Setting next_dagrun for master_dag to 2023-02-26T00:00:00+00:00, run_after=2023-02-27T00:00:00+00:00
[2023-02-27T10:02:21.819+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.090 seconds
[2023-02-27T10:02:52.186+0000] {processor.py:153} INFO - Started process (PID=6081) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T10:02:52.189+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T10:02:52.190+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:02:52.190+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T10:02:52.219+0000] {processor.py:753} INFO - DAG(s) dict_keys(['master_dag']) retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T10:02:52.254+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:02:52.254+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-27T10:02:52.286+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:02:52.286+0000] {dag.py:3441} INFO - Setting next_dagrun for master_dag to 2023-02-26T00:00:00+00:00, run_after=2023-02-27T00:00:00+00:00
[2023-02-27T10:02:52.303+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.124 seconds
[2023-02-27T10:03:22.819+0000] {processor.py:153} INFO - Started process (PID=6112) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T10:03:22.823+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T10:03:22.824+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:03:22.824+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T10:03:22.845+0000] {processor.py:753} INFO - DAG(s) dict_keys(['master_dag']) retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T10:03:22.868+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:03:22.868+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-27T10:03:22.889+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:03:22.889+0000] {dag.py:3441} INFO - Setting next_dagrun for master_dag to 2023-02-26T00:00:00+00:00, run_after=2023-02-27T00:00:00+00:00
[2023-02-27T10:03:22.904+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.091 seconds
[2023-02-27T10:03:45.199+0000] {processor.py:153} INFO - Started process (PID=6138) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T10:03:45.201+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T10:03:45.203+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:03:45.203+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T10:03:45.211+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:03:45.209+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 15
    default_args=default_args, schedule= catchup=False
                                                ^
SyntaxError: invalid syntax
[2023-02-27T10:03:45.212+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T10:03:45.255+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.061 seconds
[2023-02-27T10:03:56.544+0000] {processor.py:153} INFO - Started process (PID=6153) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T10:03:56.552+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T10:03:56.554+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:03:56.554+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T10:03:56.566+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:03:56.564+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 15
    default_args=default_args, schedule_interval= ,catchup=False
                                                  ^
SyntaxError: invalid syntax
[2023-02-27T10:03:56.567+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T10:03:56.601+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.070 seconds
[2023-02-27T10:04:04.650+0000] {processor.py:153} INFO - Started process (PID=6166) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T10:04:04.652+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T10:04:04.653+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:04:04.653+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T10:04:04.663+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:04:04.662+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 15
    default_args=default_args, schedule_interval= "/", catchup=False
                              ^
SyntaxError: keyword argument repeated
[2023-02-27T10:04:04.664+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T10:04:04.684+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.042 seconds
[2023-02-27T10:04:12.976+0000] {processor.py:153} INFO - Started process (PID=6168) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T10:04:12.978+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T10:04:12.981+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:04:12.980+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T10:04:12.991+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:04:12.989+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 15
    default_args=default_args, schedule_interval= "*/5 ", catchup=False
                              ^
SyntaxError: keyword argument repeated
[2023-02-27T10:04:12.992+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T10:04:13.039+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.080 seconds
[2023-02-27T10:04:43.451+0000] {processor.py:153} INFO - Started process (PID=6208) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T10:04:43.454+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T10:04:43.455+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:04:43.455+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T10:04:43.465+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:04:43.463+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 15
    default_args=default_args, schedule_interval= "*/5 ", catchup=False
                              ^
SyntaxError: keyword argument repeated
[2023-02-27T10:04:43.466+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T10:04:43.492+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.047 seconds
[2023-02-27T10:05:13.824+0000] {processor.py:153} INFO - Started process (PID=6245) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T10:05:13.826+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T10:05:13.827+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:05:13.826+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T10:05:13.832+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:05:13.832+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 15
    default_args=default_args, schedule_interval= "*/5 ", catchup=False
                              ^
SyntaxError: keyword argument repeated
[2023-02-27T10:05:13.833+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T10:05:13.884+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.063 seconds
[2023-02-27T10:05:43.223+0000] {processor.py:153} INFO - Started process (PID=6275) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T10:05:43.225+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T10:05:43.227+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:05:43.227+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T10:05:43.236+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:05:43.235+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 15
    default_args=default_args, schedule_interval="*/5 * * * *", catchup=False
                              ^
SyntaxError: keyword argument repeated
[2023-02-27T10:05:43.237+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T10:05:43.265+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.049 seconds
[2023-02-27T10:06:13.679+0000] {processor.py:153} INFO - Started process (PID=6315) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T10:06:13.681+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T10:06:13.682+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:06:13.682+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T10:06:13.689+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:06:13.688+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 15
    default_args=default_args, schedule_interval="*/5 * * * *", catchup=False
                              ^
SyntaxError: keyword argument repeated
[2023-02-27T10:06:13.690+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T10:06:13.710+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.035 seconds
[2023-02-27T10:06:44.113+0000] {processor.py:153} INFO - Started process (PID=6342) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T10:06:44.116+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T10:06:44.118+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:06:44.117+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T10:06:44.141+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:06:44.136+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/master_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/master_dag.py", line 15
    default_args=default_args, schedule_interval="*/5 * * * *", catchup=False
                              ^
SyntaxError: keyword argument repeated
[2023-02-27T10:06:44.143+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T10:06:44.167+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.061 seconds
[2023-02-27T10:06:52.266+0000] {processor.py:153} INFO - Started process (PID=6355) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T10:06:52.269+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T10:06:52.271+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:06:52.270+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T10:06:52.327+0000] {processor.py:753} INFO - DAG(s) dict_keys(['master_dag']) retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T10:06:52.398+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:06:52.398+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-27T10:06:52.422+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:06:52.422+0000] {dag.py:3441} INFO - Setting next_dagrun for master_dag to 2023-02-27T10:00:00+00:00, run_after=2023-02-27T10:05:00+00:00
[2023-02-27T10:06:52.451+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.194 seconds
[2023-02-27T10:07:22.672+0000] {processor.py:153} INFO - Started process (PID=6382) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T10:07:22.674+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T10:07:22.676+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:07:22.676+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T10:07:22.698+0000] {processor.py:753} INFO - DAG(s) dict_keys(['master_dag']) retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T10:07:22.724+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:07:22.724+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-27T10:07:22.748+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:07:22.748+0000] {dag.py:3441} INFO - Setting next_dagrun for master_dag to 2023-02-27T10:00:00+00:00, run_after=2023-02-27T10:05:00+00:00
[2023-02-27T10:07:22.770+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.102 seconds
[2023-02-27T10:07:53.089+0000] {processor.py:153} INFO - Started process (PID=6420) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T10:07:53.091+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T10:07:53.093+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:07:53.093+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T10:07:53.122+0000] {processor.py:753} INFO - DAG(s) dict_keys(['master_dag']) retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T10:07:53.150+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:07:53.150+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-27T10:07:53.177+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:07:53.177+0000] {dag.py:3441} INFO - Setting next_dagrun for master_dag to 2023-02-27T10:05:00+00:00, run_after=2023-02-27T10:10:00+00:00
[2023-02-27T10:07:53.194+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.110 seconds
[2023-02-27T10:08:23.458+0000] {processor.py:153} INFO - Started process (PID=6459) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T10:08:23.460+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T10:08:23.461+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:08:23.461+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T10:08:23.487+0000] {processor.py:753} INFO - DAG(s) dict_keys(['master_dag']) retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T10:08:23.519+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:08:23.519+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-27T10:08:23.549+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:08:23.549+0000] {dag.py:3441} INFO - Setting next_dagrun for master_dag to 2023-02-27T10:05:00+00:00, run_after=2023-02-27T10:10:00+00:00
[2023-02-27T10:08:23.586+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.133 seconds
[2023-02-27T10:08:53.813+0000] {processor.py:153} INFO - Started process (PID=6487) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T10:08:53.815+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T10:08:53.816+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:08:53.816+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T10:08:53.834+0000] {processor.py:753} INFO - DAG(s) dict_keys(['master_dag']) retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T10:08:53.856+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:08:53.856+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-27T10:08:53.884+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:08:53.884+0000] {dag.py:3441} INFO - Setting next_dagrun for master_dag to 2023-02-27T10:05:00+00:00, run_after=2023-02-27T10:10:00+00:00
[2023-02-27T10:08:53.909+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.099 seconds
[2023-02-27T10:09:24.263+0000] {processor.py:153} INFO - Started process (PID=6526) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T10:09:24.265+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T10:09:24.267+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:09:24.267+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T10:09:24.305+0000] {processor.py:753} INFO - DAG(s) dict_keys(['master_dag']) retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T10:09:24.333+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:09:24.333+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-27T10:09:24.374+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:09:24.374+0000] {dag.py:3441} INFO - Setting next_dagrun for master_dag to 2023-02-27T10:05:00+00:00, run_after=2023-02-27T10:10:00+00:00
[2023-02-27T10:09:24.391+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.133 seconds
[2023-02-27T10:09:54.687+0000] {processor.py:153} INFO - Started process (PID=6553) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T10:09:54.691+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T10:09:54.693+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:09:54.692+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T10:09:54.721+0000] {processor.py:753} INFO - DAG(s) dict_keys(['master_dag']) retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T10:09:54.745+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:09:54.745+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-27T10:09:54.765+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:09:54.765+0000] {dag.py:3441} INFO - Setting next_dagrun for master_dag to 2023-02-27T10:05:00+00:00, run_after=2023-02-27T10:10:00+00:00
[2023-02-27T10:09:54.776+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.094 seconds
[2023-02-27T10:10:25.084+0000] {processor.py:153} INFO - Started process (PID=6596) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T10:10:25.087+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T10:10:25.090+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:10:25.090+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T10:10:25.121+0000] {processor.py:753} INFO - DAG(s) dict_keys(['master_dag']) retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T10:10:25.145+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:10:25.145+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-27T10:10:25.170+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:10:25.169+0000] {dag.py:3441} INFO - Setting next_dagrun for master_dag to 2023-02-27T10:10:00+00:00, run_after=2023-02-27T10:15:00+00:00
[2023-02-27T10:10:25.183+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.108 seconds
[2023-02-27T10:10:55.551+0000] {processor.py:153} INFO - Started process (PID=6637) to work on /opt/airflow/dags/master_dag.py
[2023-02-27T10:10:55.555+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/master_dag.py for tasks to queue
[2023-02-27T10:10:55.556+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:10:55.556+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/master_dag.py
[2023-02-27T10:10:55.595+0000] {processor.py:753} INFO - DAG(s) dict_keys(['master_dag']) retrieved from /opt/airflow/dags/master_dag.py
[2023-02-27T10:10:55.647+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:10:55.647+0000] {dag.py:2690} INFO - Sync 1 DAGs
[2023-02-27T10:10:55.680+0000] {logging_mixin.py:137} INFO - [2023-02-27T10:10:55.680+0000] {dag.py:3441} INFO - Setting next_dagrun for master_dag to 2023-02-27T10:10:00+00:00, run_after=2023-02-27T10:15:00+00:00
[2023-02-27T10:10:55.698+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/master_dag.py took 0.159 seconds
